okhttp/src/jvmMain/kotlin/okhttp3/ConnectionSpec.kt

```diff
@@ -191,8 +209,8 @@ class ConnectionSpec internal constructor(
     if (!isTls) return "ConnectionSpec()"
 
     return ("ConnectionSpec(" +
-        "cipherSuites=${Objects.toString(cipherSuites, "[all enabled]")}, " +
-        "tlsVersions=${Objects.toString(tlsVersions, "[all enabled]")}, " +
+        "cipherSuites=${cipherSuitesAsString?.let { it.contentToString() } ?: "[all enabled]"}, " +
+        "tlsVersions=${tlsVersionsAsString?.let { it.contentToString() } ?: "[all enabled]"}, " +
```

Comment:
I'm not sure how much we care, but the test below indicates it is behaviour changing.